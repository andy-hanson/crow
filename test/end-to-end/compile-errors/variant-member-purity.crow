main void()
	info log "done"

v variant
m record mut, v variant-member
