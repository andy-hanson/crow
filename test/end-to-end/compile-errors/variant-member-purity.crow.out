exit code: 0
stdout:
info: done
stderr:
test/end-to-end/compile-errors/variant-member-purity.crow 5:15-5:31 Variant 'v' has purity 'data', but member 'm' has purity 'mut'.
